webdriver/tests/element_send_keys/form_controls.py

```diff
@@ -42,6 +43,20 @@ def test_input_append(session):
     assert element.property("value") == "abc"
 
 
+def test_input_append_with_focus(session):
+    session.url = inline("<input value=a>")
+    element = session.find.css("input", all=False)
+    assert element.property("value") == "a"
+
+    element_send_keys(session, element, "b")
+    assert element.property("value") == "ab"
```

Comment:
Just set `ab` as value directly, so that we don't need those two lines.

Reply:
Sorry for the delay, I missed your comment. I'm doing this specifically to verify that even though a is already there (and verified), when I send "b" via sendkeys, the letter "a" is not deleted.

Reply:
I see. So give that this test is about appending, can you move out the case for inserting, or modify the test name?